compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/test/conv_to_im2col.mlir

```diff
@@ -453,3 +453,75 @@ util.func public @conv_1d_nhc_chf(%arg0: tensor<1x3x2xf32>, %arg1: tensor<2x2x2x
 // CHECK-SAME:   input_k_perm = [1, 0]
 // CHECK-SAME:   ins({{.*}} : tensor<1x3x2xf32>)
 // CHECK-SAME:   outs({{.*}} : tensor<1x2x4xf32>) -> tensor<1x2x4xf32>
+
+// -----
+
+util.func public @conv_2d_nhwgc_gfhwc(%arg0: tensor<1x10x10x7x4xf32>, %arg1: tensor<7x16x3x3x4xf32>, %arg2: tensor<1x8x8x7x16xf32>) -> tensor<1x8x8x7x16xf32> {
+  %0 = linalg.conv_2d_nhwgc_gfhwc
+    {dilations = dense<1> : tensor<2xi64>, strides = dense<1> : tensor<2xi64> }
+     ins(%arg0, %arg1: tensor<1x10x10x7x4xf32>, tensor<7x16x3x3x4xf32>)
+    outs(%arg2: tensor<1x8x8x7x16xf32>) -> tensor<1x8x8x7x16xf32>
+  util.return %0 : tensor<1x8x8x7x16xf32>
+}
+// CHECK-DAG:  #[[LHS_MAP:.+]] = affine_map<(d0, d1, d2, d3, d4, d5) -> (d3, d0, d1, d2, d5)>
+// CHECK-DAG:  #[[RHS_MAP:.+]] = affine_map<(d0, d1, d2, d3, d4, d5) -> (d3, d4, d5)>
+// CHECK-DAG:  #[[OUT_MAP:.+]] = affine_map<(d0, d1, d2, d3, d4, d5) -> (d0, d1, d2, d3, d4)>
+// CHECK:      util.func public @conv_2d_nhwgc_gfhwc(
+// CHECK-SAME:   %[[IMG:.+]]: [[IMG_T:tensor<1x10x10x7x4xf32>]]
+// CHECK-SAME:   %[[FIL:.+]]: [[FIL_T:tensor<7x16x3x3x4xf32>]]
+// CHECK-SAME:   %[[OUT:.+]]: [[OUT_T:tensor<1x8x8x7x16xf32>]]
+// CHECK:      %[[EMPTY:.+]] = tensor.empty() : [[LHS_T:tensor<7x1x8x8x36xf32>]]
```

Comment:
Does it require the `g dimension` to be in front? Does it work when the `n dimension` is larger than 1? It's better to modify either of these two tests to cover a case that n>1.